src/doctopus/db/schema.clj

```diff
@@ -87,8 +89,20 @@
 (defn- create-fts-document-index
   [db-name]
   (let [update-sql "UPDATE documents SET search_vector = to_tsvector('english', name || ' ' || body)"
-        idx-sql "CREATE INDEX fts_idx ON documents USING GIN(search_vector)"]
-    (log/info "Creating FTS Index for the documents table")
+        idx-sql (->> ["DO $$"
```

Comment:
I think moving all the raw SQL in to yesql sounds like a lovely improvement. For now, everywhere we _are_ using Korma is veeeeeery simple and inoffensive, so I'm inclined to leave it in place. (Besides, despite yasql's author's grousing, Korma is _pretty nice_ for little things.)